Source/DafnyStandardLibraries/examples/WrappersExamples.dfy

```diff
@@ -1,4 +1,4 @@
-module Demo {
+module WrappersExamples {
```

Comment:
Conflicts with #4713, and I think I prefer that PR's approach of just using the default module.